templates/set-lcm-mode.ps1.erb

```diff
@@ -0,0 +1,51 @@
+Write-Host "Setting LCM"
+$LCMRefreshMode = '<%= @refresh_mode %>'
+$generationDate = (get-date -format "M/d/yyyy HH:mm:ss")
+$encoding       = [system.Text.Encoding]::UTF8
+$Namespace      = 'root/Microsoft/Windows/DesiredStateConfiguration'
+$ClassName      = 'MSFT_DSCLocalConfigurationManager'
+
+Write-Verbose "LCM Mode: $LCMRefreshMode"
```

Comment:
The param set at line 42 sets verbose for the Invoke-CimMethod at line 51, so verbose messages show up. Line 1 should be a verbose message as well instead of write-host, but both were added during debugging and really don't add much value otherwise so it may be best just to remove both.